ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/IWhisperService.cs

```diff
@@ -143,6 +143,11 @@ public struct WhisperList
         /// </summary>
         public WhisperConfig Config;
 
+        /// <summary>
+        /// Markdown formatted string displayed at the top of the form.
+        /// </summary>
+        public string Markdown;
```

Comment:
I think this also requires a corresponding change to `ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/WhisperFormBuilder.cs:26`

Reply:
If this is a change for just WhisperList, would I need to update the form builder?

Reply:
Yes; the `WhisperFormBuilder` also has a `BuildRequest(WhisperList list, Proto.Session session)` method where the message is constructed.